core/templates/dev/head/pages/practice-session-page/practice-session-page.controller.ts

```diff
@@ -57,8 +57,16 @@ oppia.directive('practiceSessionPage', ['UrlInterpolationService', function(
               topic_name: ctrl.topicName
             });
           $http.get(practiceSessionsDataUrl).then(function(result) {
+            var skillList = [];
+            var skillDescriptions = [];
+            for (var skill in result.data.skills_with_description) {
```

Comment:
skill -> skillId

Reply:
Do you mean skillId for the variable in the loop? We're iterating through skill objects so that's why I kept it as skill

Reply:
But in the backend dict, the key of skills_with_description was skill_id right?

Reply:
Aah okay I understand what you mean! Yep the key is a skill id..I'll make those changes